pkg/registry/generic/registry/store.go

```diff
@@ -252,6 +272,15 @@ func (e *Store) Create(ctx api.Context, obj runtime.Object) (runtime.Object, err
 		if accessor.GetDeletionTimestamp() != nil {
 			msg := &err.(*kubeerr.StatusError).ErrStatus.Message
 			*msg = fmt.Sprintf("object is being deleted: %s", *msg)
+			// TODO: remove this block after 1.6
+			userAgent, _ := api.UserAgentFrom(ctx)
+			if !isOldKubectl(userAgent) {
+				return nil, err
+			}
+			if e.QualifiedResource.Resource != "replicationcontrollers" {
```

Comment:
OK, this new review thingy is strange - I see @lavalamp's reply at the bottom.

I don't know - I thought that replica set is just a copy of replication controller + new selector.